hadoop-tools/hadoop-distcp/src/main/java/org/apache/hadoop/tools/DistCp.java

```diff
@@ -438,6 +446,15 @@ protected DistCpContext getContext() {
     return context;
   }
 
+  /**
+   * Sets the current context.
+   *
+   * @param context context to be set to.
+   */
+  public void setContext(DistCpContext context) {
```

Comment:
is this for testing or part of the public API?

Reply:
Testing on my deployment, removed and reverted the relevant parts.